docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -4,7 +4,7 @@ A Service Locator is embedded in Lagom's development environment, allowing servi
 
 ## Default port
 
-By default, the service locator runs on port `8000`, but it is possible to use a different port. For instance, you can tell the service locator to run on port 10000 by adding the following to your build.
+By default, the service locator runs on port `9008` (historically it was `8000`), but it is possible to use a different port. For instance, you can tell the service locator to run on port `10000` by adding the following to your build.
```

Comment:
same as above, we don't need to mention that we used to use another port in the past